.github/workflows/branch_pages_deploy.yml

```diff
@@ -17,38 +17,71 @@ jobs:
         with:
           node-version-file: '.nvmrc'
 
-      - name: 'Set BRANCH_NAME variable'
-        run: |
-          echo "BRANCH_NAME=${GITHUB_REF##*/}" >> $GITHUB_ENV
+      - name: 'Install dependencies'
+        run: npm ci
+
+      - name: 'Save PR data'
+        uses: actions/github-script@v6
+        id: prData
+        with:
+          script: |
+            const getData = require('./.github/workflow-scripts/gh-pages/get-pr-data.js');
+            return await getData({
+              pullNumber: ${{ github.event.inputs.pullNumber }},
+              github,
+              context
+            });
+
+      - name: 'Download artefacts from PR'
```

Comment:
I think we use the 'artifact' spelling elsewhere

Reply:
Updated 👍